src/pages/ProductDetail/ProductHeadSection.js

```diff
@@ -0,0 +1,225 @@
+import React from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const ProductHeadSection = ({ product }) => {
+  var settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+  return (
+    <HeadSection>
+      <HeadImg className="wrapper">
+        <Slider {...settings} className="imgSlider">
+          {product.image &&
+            product.image.map(image => (
+              <div>
+                <div>
+                  <img alt="상품 사진" src={image} />
+                </div>
+              </div>
+            ))}
+          {/* <div>
+            <div>
+              <img alt="상품 사진" src={product.image && product.image[0]} />
+            </div>
+          </div>
+          <div>
+            <div>
+              <img alt="빵사진" src={product.image && product.image[1]} />
+            </div>
+          </div>
+          <div>
+            <div>3333333333</div>
+          </div> */}
+        </Slider>
+      </HeadImg>
+      <HeadInfo>
+        <InfoWrap>
+          <InfoTitle>{product && product.title}</InfoTitle>
+          <InfoPrice>
+            {product.discount_percent !== 0 && (
+              <PricePercent>
+                {product && product.discount_percent}%
+              </PricePercent>
+            )}
+            {product.discount_percent !== 0 && (
+              <PriceDiscount>
+                {product.price && product.discounted_price.toLocaleString()}원
+              </PriceDiscount>
+            )}
+            <PriceOriginal>
+              {product.price && product.price.toLocaleString()}원
+            </PriceOriginal>
+          </InfoPrice>
+        </InfoWrap>
+        <InfoHost>
+          <HostImage>
+            <HostImg
+              alt="호스트 이미지"
+              src={product.user && product.user.image}
+            />
+          </HostImage>
+          <HostContent>
+            <HostId>{product.user && product.user.name}</HostId>
+            <HostDetail>
+              프립 {product.user && product.user.product_count}
+              <span>|</span>
+              후기 {product.user && product.user.review_count}
+              {/* <span>|</span>
+              "저장 " "13" */}
+            </HostDetail>
+          </HostContent>
+        </InfoHost>
```

Comment:
```suggestion
        {product.user && (
          <InfoHost>
            <HostImage>
              <HostImg alt="호스트 이미지" src={product.user.image} />
            </HostImage>
            <HostContent>
              <HostId>{product.user.name}</HostId>
              <HostDetail>
                프립 {product.user.product_count}
                <span>|</span>
                후기 {product.user.review_count}
              </HostDetail>
            </HostContent>
          </InfoHost>
        )}
```
리뷰 참고해서 중복되는 조건을 줄여보세요!

Reply:
반영하였습니다!